### PR TITLE
Add emsPublisherUrlSuffix as a config value

### PR DESCRIPTION
**Pull request by radufa**
**Description**
For publishing events to EMS, a suffix must be added to the EMS URL obtained from Service Key.
The value of the suffix is made configurable

Changes proposed in this pull request:

- Add a new configurable value for EMS URL suffix

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/10317

### CODE REVIEW

File: resources/eventing/charts/event-publisher-proxy/values.yaml [hunk 1]
```diff
@@ -29,4 +29,6 @@ beb:
   config:
    # eventTypePrefix is the prefix of the eventType as per BEB specification
    eventTypePrefix: sap.kyma.custom
+   # emsPublisgerUrlSuffix is the suffix added to EMS url obtined from service key
```
Comment (marcobebway) [inline resources/eventing/charts/event-publisher-proxy/values.yaml:32]: Typos:
- emsPublisgerUrlSuffix
- obtined

Comment (radufa) [reply resources/eventing/charts/event-publisher-proxy/values.yaml]: Done

**Review by marcobebway — APPROVED** [review]

**Pull request merged**